mne/io/base.py

```diff
@@ -816,6 +817,22 @@ def get_data(self, picks=None, start=0, stop=None,
             'bad' are omitted. If 'NaN', the bad samples are filled with NaNs.
         return_times : bool
             Whether to return times as well. Defaults to False.
+        units : str | dict | None
+            Specify the unit(s) that the data should be returned in. If
+            ``None`` (default), the data is returned in the
+            channel-type-specific default units, which are SI units (see
+            :ref:`units` and :term:`data channels`). If a string, must be an SI
+            unit that will be used to scale the data from all channels of the
```

Comment:
I feel that we should change the wording here @sappelhoff, because one may understand that they need to use SI units (as defined with their SI prefixes). What do you think? Any suggestions on how to make this clearer?

Reply:
See 9e5ea86